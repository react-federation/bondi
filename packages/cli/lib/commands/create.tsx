import React from 'react'
import { render } from 'ink'
import FormBuilder from '../components/FormBuilder'
import remoteConfig from '../forms/remotes.form'

const host = () => {
  render(<FormBuilder form={remoteConfig as any} />)
}

export default host
