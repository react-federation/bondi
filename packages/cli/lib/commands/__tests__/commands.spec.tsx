import { render } from 'ink-testing-library'
import create from '../create'

function setup() {
  return render(create as any)
}

describe('command create', () => {
  test('should match host snapshot', () => {
    expect(setup()).toMatchSnapshot()
  })
})
