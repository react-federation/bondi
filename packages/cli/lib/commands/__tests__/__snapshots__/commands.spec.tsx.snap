// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`command create should match host snapshot 1`] = `
Object {
  "cleanup": [Function],
  "frames": Array [
    "",
  ],
  "lastFrame": [Function],
  "rerender": [Function],
  "stderr": Stderr {
    "_events": Object {},
    "_eventsCount": 0,
    "_maxListeners": undefined,
    "frames": Array [],
    "lastFrame": [Function],
    "write": [Function],
    Symbol(kCapture): false,
  },
  "stdin": Stdin {
    "_events": Object {},
    "_eventsCount": 0,
    "_maxListeners": undefined,
    "isTTY": true,
    "write": [Function],
    Symbol(kCapture): false,
  },
  "stdout": Stdout {
    "_events": Object {
      "resize": [Function],
    },
    "_eventsCount": 1,
    "_lastFrame": "",
    "_maxListeners": undefined,
    "frames": Array [
      "",
    ],
    "lastFrame": [Function],
    "write": [Function],
    Symbol(kCapture): false,
  },
  "unmount": [Function],
}
`;

exports[`command create should match remote snapshot 1`] = `
Object {
  "cleanup": [Function],
  "frames": Array [
    "",
  ],
  "lastFrame": [Function],
  "rerender": [Function],
  "stderr": Stderr {
    "_events": Object {},
    "_eventsCount": 0,
    "_maxListeners": undefined,
    "frames": Array [],
    "lastFrame": [Function],
    "write": [Function],
    Symbol(kCapture): false,
  },
  "stdin": Stdin {
    "_events": Object {},
    "_eventsCount": 0,
    "_maxListeners": undefined,
    "isTTY": true,
    "write": [Function],
    Symbol(kCapture): false,
  },
  "stdout": Stdout {
    "_events": Object {
      "resize": [Function],
    },
    "_eventsCount": 1,
    "_lastFrame": "",
    "_maxListeners": undefined,
    "frames": Array [
      "",
    ],
    "lastFrame": [Function],
    "write": [Function],
    Symbol(kCapture): false,
  },
  "unmount": [Function],
}
`;
